lxd/certificates.go

```diff
@@ -439,6 +440,11 @@ func certificatesPost(d *Daemon, r *http.Request) response.Response {
 		return response.NotImplemented(fmt.Errorf("Password authentication is no longer supported, please update your client"))
 	}
 
+	// Validate name.
+	if strings.HasPrefix(req.Name, "-") || strings.Contains(req.Name, "/") {
+		return response.BadRequest(fmt.Errorf("Invalid certificate name"))
```

Comment:
Could you please add some more information to the error message so that the user will know what they need to change?

E.g. `fmt.Errorf("Invalid certificate name %q, name must not start with a hyphen or contain forward slashes")`